.vscode/settings.json

```diff
@@ -1,7 +1,5 @@
 {
-  "eslint.workingDirectories": [
-    {
-      "mode": "auto"
+    "editor.codeActionsOnSave":{
+      "source.organizeImports.biome": "explicit"
```

Comment:
I took the liberty of adding an auto-sort imports on save. I think this is handy, but feel free to disagree :-)

Reply:
BTW I noticed that biome respects grouped imports, i.e. there is no sorting off import statements separated by empty lines

Reply:
Also imports from the same module that are on different lines are not sorted or combined. But once you combine them, they will be sorted alphabetically 🤔

Reply:
cool thanks, the organize sort is not part of lint or format commands so its nice the editor can do it.